content/articles/getting-started-with-nuxt-3/index.md

```diff
@@ -0,0 +1,135 @@
+
+With other tool parts upgrades, Webpack 5, Vite, ESBuild, and PostCSS are now supported. This means that both development and production are built remarkably quickly.
+
+#### Nitro Engine
+Nitro engine is the main feature of Nuxt 3. It is Nuxt 3's new server engine.
+
+The Nitro engine gives Nuxt applications a lot of additional capabilities and features. It also boosts Nuxt 3's performance and improvements.
+
+#### Nuxt Bridge
+The Nuxt Bridge is a compatibility layer that enables you to upgrade from Nuxt 2 to Nuxt 3. It enables you to access the new Nuxt 3 features on an existing Nuxt 2 application. The new set of features includes:
++ Nitro engine.
++ Dev tools.
++ Nuxt CLI.
++ Built-in TypeScript and ES modules Support.
++ Vite integration
+
+You can learn more about the [Nuxt Bridge here](https://v3.nuxtjs.org/getting-started/bridge/).
+
+#### Vue 3 and Vite support
+Nuxt 3 was created to work with Vue 3. You can use features like the Composition API, enhanced module imports, and faster overall app speeds because Nuxt 3 is written in Vue 3.
+
+#### TypeScript
+Nuxt 3 is written in TypeScript for easier maintenance and built-in Typescript support.
+
+Nuxt 3 adopts TypeScript and ES Modules (ESM) to enhance the development experience. TypeScript will prevent runtime errors and enhance auto-completion.
+
+However, it's not a must to write your Nuxt 3 application in TypeScript (using JavaScript is fine and works as expected). You'll be able to use Nuxt 3 all the same with no problems.
+
+#### New file structure
+The `app.vue` file act as the primary element of your Nuxt 3 application. The `pages/` directory is optional in Nuxt 3, and you can work with a single page, `app.vue`. Everything you add to `app.vue`, such as JavaScript and CSS, will be shared across all pages. 
+
+#### Cross-platform support
+Over the years, JavaScript has come a long way, and there are now a number of platforms that use it. Nuxt 3 will run smoothly in any JS environment. This includes Node.js, Cloudflare Workers, and Deno.
+
+### Nuxt 3 plugins
+Plugins in Nuxt allow you to extend your applications' functionalities with the JavaScript library. With Nuxt 3 plugins, you can extend the functionality of your Nuxt application in three different ways:
+ + Making the functions to be available on the client-side and server-side.
+ + Using your preferred Vue.js plugins in your Nuxt application.
+ + Using external javascript packages and modules into your Nuxt application.
+
+ You can learn more about [plugins here](https://v3.nuxtjs.org/docs/directory-structure/plugins/).
+
+### Why you need to use Nuxt 3
+Here are some of the main reasons Nuxt 3 is a great framework and why you should use it in your next project.
+
+1. Flexibility.
+Nuxt allows you to display your application in three different ways.
```

Comment:
This should be:
1. Flexibility - Nuxt allows you to display your application in three different ways.

Or:
1. Flexibility: Nuxt allows you to display your application in three different ways.

Fix the ones after it too.